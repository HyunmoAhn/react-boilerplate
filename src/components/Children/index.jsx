import React from 'react';
import workImage from 'assets/image/work-image.jpg';
import './Children.scss';

function Children() {
	return(
		<div>
			Children
			<img src={workImage} alt="work-image" />
		</div>
	);
}

export default Children;