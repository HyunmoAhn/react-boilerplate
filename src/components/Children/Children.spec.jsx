import React from 'react';
import { shallow } from 'enzyme';

import Children from './';
describe('<Children />', () => {
	it('should match to snapshot when render default', () => {
		const wrapper = shallow(<Children />);

		expect(wrapper).toMatchSnapshot();
	});
});