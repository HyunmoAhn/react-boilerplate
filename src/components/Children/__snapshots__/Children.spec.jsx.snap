// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Children /> should match to snapshot when render default 1`] = `
<div>
  Children
  <image
    alt="work-image"
    src={Object {}}
  />
</div>
`;
